index.js

```diff
@@ -30,7 +30,7 @@ const config = {
 		'object-shorthand': 'error',
 		'prefer-const': 'error',
 		'quote-props': ['error', 'as-needed'],
-		'sort-keys': ['error', 'asc', {caseSensitive: false, natural: true}],
+		'sort-keys': ['error', 'asc', {caseSensitive: true, natural: true}],
```

Comment:
`true` is the default, so we could get rid of this... but maybe it's good to have it here so we don't have to dig for the default again... 🤷‍♂

Reply:
Yeah, no harm done having it in there.